src/assets/i18n/en_US.json

```diff
@@ -229,111 +229,111 @@
+  "account.orderdetails.links.continue_shopping": "Continue shopping",
+  "account.orderdetails.links.return_to_orders": "Back to orders",
+  "account.orderdetails.order_number.label": "Order no.",
+  "account.orderdetails.order_status.label": "Order status",
   "account.orderdetails.print_link.text": "Print",
   "account.orderlist.no_orders_message": "Currently there are no orders.",
   "account.orderlist.no_placed_orders_message": "You have not placed an order yet.",
   "account.orderlist.table.buyer": "Buyer",
-  "account.orderlist.table.date_of_order": "Order Date",
+  "account.orderlist.table.date_of_order": "Order date",
   "account.orderlist.table.destination": "Destination",
   "account.orderlist.table.items": "Items",
-  "account.orderlist.table.order_number": "Order No.",
+  "account.orderlist.table.order_number": "Order no.",
   "account.orderlist.table.order_status": "Status",
-  "account.orderlist.table.order_total": "Order Total",
-  "account.ordertemplates.heading": "Order Templates",
-  "account.ordertemplates.link": "Order Templates",
-  "account.ordertemplates.widget.heading": "My Order Templates",
-  "account.ordertemplates.widget.view_all.link": "View All Order Templates",
-  "account.organization.cost_center_management": "Cost Center Management",
-  "account.organization.cost_center_management.back_to_list": "Back to Cost Center Management",
+  "account.orderlist.table.order_total": "Order total",
+  "account.ordertemplates.heading": "Order templates",
+  "account.ordertemplates.link": "Order templates",
+  "account.ordertemplates.widget.heading": "My order templates",
+  "account.ordertemplates.widget.view_all.link": "View all order templates",
+  "account.organization.cost_center_management": "Cost center management",
+  "account.organization.cost_center_management.back_to_list": "Back to cost center management",
   "account.organization.cost_center_management.buyer.add.confirmation": "{{0, plural, one{# buyer has been added to the cost center.} other{# buyers have been added to the cost center.}}}",
   "account.organization.cost_center_management.buyer.remove.confirmation": "The user \"{{0}}\" has been removed from the cost center.",
   "account.organization.cost_center_management.buyer.update.confirmation": "The budget of user \"{{0}}\" has been updated.",
   "account.organization.cost_center_management.create.confirmation": "The cost center \"{{0}}\" has been created.",
   "account.organization.cost_center_management.delete.confirmation": "The cost center has been deleted.",
   "account.organization.cost_center_management.update.confirmation": "The cost center \"{{0}}\" has been updated.",
-  "account.organization.user_management": "User Management",
-  "account.organization.user_management.back_to_list": "Back to User Management",
+  "account.organization.user_management": "User management",
+  "account.organization.user_management.back_to_list": "Back to user management",
```

Comment:
Is User Management a product/proper name??

Reply:
No.